arch/arm/src/lpc17xx_40xx/hardware/lpc17_40_wdt.h

```diff
@@ -89,6 +89,8 @@
 
 /* Watchdog feed sequence register  */
 
+#define WDT_FEED_KEY_1                (0xAA)    /* Watchdog feed key 1 */
```

Comment:
please change to 0xaa (NuttX coding style uses hexacode in lower case)

Reply:
Oh, I wasn't aware of this.  
Do I need a new PR, or can I force push to a merged PR?

Reply:
I suggest you to send a new PR to fix it. Thank you!